client/lib/widgets/side_menu.dart

```diff
@@ -27,42 +27,46 @@ class SideMenu extends StatelessWidget {
             ),
             Divider(),
             InkWell(
-              onTap: (){
-                Navigator.of(context).pushNamedAndRemoveUntil(
-                        routes.homePage, (route) => false);
+              onTap: () {
+                Navigator.of(context)
+                    .pushNamedAndRemoveUntil(routes.homePage, (route) => false);

```

Comment:
all the navigator calls here should just be "pushNamed" instead of `pushNamedAndRemoveUntil`, no need to purge the history of paths when navigating a menu and that'd be weird behavior on web.

Reply:
true but that would produce weird behavior when navigating from home page to home page since its possible the way it's set up now

Reply:
i guess we should also do a check to make sure users aren't navigating to the same page again and just not push in that case.
either way since it's crunch time it's not as relevant, will rework this at some point way later